Pipelines/Templates/deploy-Solution.yml

```diff
@@ -295,4 +295,56 @@ steps:
     makerPortalUrl: '$(TestAutomationMakerPortalUrl)'
     loginMethod: '$(TestAutomationLoginMethod)'
 
+- template: Hooks\deploy-power-page-pre-hook.yml
+
+- pwsh: |
```

Comment:
@RajeevPentyala Is this just debug code that we can remove?

Reply:
@mikefactorial Removed the code.